webapp/app.py

```diff
@@ -178,17 +178,39 @@ def _get_contributors():
     return contributors
 
 
-def _filter_contributors(contributors):
-    # Distinguish team_members from contributors
-
-    member_usernames = [member["login"] for member in TEAM_MEMBERS]
+def _filter_team_members_from_contributors(contributors):
+    member_usernames = {member["login"] for member in TEAM_MEMBERS}
     return [
         contributor
         for contributor in contributors
         if contributor["login"] not in member_usernames
     ]
 
 
+def _is_not_renovate_bot(contributor):
+    # Using "login" to check for renovate-bot as
+    # GitHub treats it as a user, i.e., type: "User"
+    
+    return contributor["login"] != "renovate-bot"
+
+
+def _filter_bots_from_contributors(contributors):
+    return [
+        contributor
+        for contributor in contributors
+        if (
+            contributor["type"].lower() != "bot" 
+            and _is_not_renovate_bot(contributor)
```

Comment:
We only have one case of this currently, so I don't think we need to create a whole extra function for special-case bots yet; how about something like this?

```suggestion
            and contributor["id"] != 25180681 # renovate-bot
```